integration/cluster.go

```diff
@@ -80,7 +88,7 @@ func (c *cluster) fillClusterForMembers() error {
 	addrs := make([]string, 0)
 	for _, m := range c.Members {
 		scheme := "http"
-		if !m.PeerTLSInfo.Empty() {
+		if m.PeerTLSInfo != nil {
```

Comment:
I want an empty tls to fail through the regular tls machinery; if a tls is given it should have cert data.